src/tests/ftest/util/test_utils_pool.py

```diff
@@ -603,7 +603,7 @@ def exclude(self, ranks, tgt_idx=None):
             CmdResult: Object that contains exit status, stdout, and other information.
 
         """
-        return self.dmg.pool_exclude(self.identifier, ranks, tgt_idx)
+        return self.dmg.pool_exclude(self.identifier, ranks, tgt_idx, force=True)
```

Comment:
This maintains previous behavior, but we should have some test to verify force=False behaves as expected